jcommune-view/jcommune-web-view/src/main/webapp/resources/javascript/app/pollNewTopic.js

```diff
@@ -0,0 +1,53 @@
+$(document).ready(function() {
+
+    if ($("div.pollItemsValue").length == maxPollItems) {
+        $("a#add").hide();
+    }
+
+    $("#addPollButton").click(function(){
```

Comment:
this button have 2 action? delete and add? if yes then id #addPollButton is not appropriate.